### PR TITLE
PF-810 add google storage transfer servivce

### PR DESCRIPTION
**Pull request by jaycarlton**
Enable the Google Storage Transfer service, to be used in GCS Bucket Clone operations.

### CODE REVIEW

File: terra-workspace-manager/api-services.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module "enable-services" {
-  source      = "github.com/broadinstitute/terraform-shared.git//terraform-modules/api-services?ref=services-0.3.0-tf-0.12"
+  source      = "github.com/broadinstitute/terraform-shared.git//terraform-modules/api-services?ref=jaycarlton/PF-810"
```
Comment (wchamber) [inline terra-workspace-manager/api-services.tf]: I don't think you want to change this even in this draft unless you're planning on modifying the github.com/broadinstitute/terraform-modules repo, which shouldn't be necessary.

**Review by wchamber — APPROVED** [review]
Consider adding something to the PR description about why we're doing this.
Make sure to follow the https://github.com/broadinstitute/terraform-ap-deployments#workflow on how to correctly dance between commits to these two repos.

File: terra-workspace-manager/api-services.tf (hunk 1 above)
Comment (jaycarlton) [reply terra-workspace-manager/api-services.tf]: You're right; that branch doesn't even exist (hopefully) in that other repo.

File: terra-env/main.tf [hunk 2]
```diff
@@ -72,7 +72,7 @@ module "sam" {
 }
 
 module "workspace_manager" {
-  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=terra-workspace-manager-0.6.6"
+  source = "github.com/broadinstitute/terraform-ap-modules.git//terra-workspace-manager?ref=jaycarlton%2FPF-810"
```
Comment (yonghaoy) [inline terra-env/main.tf]: Minor suggestion: personally I did not use `/ ` in branch name, but use `-` instead.

**Review by yonghaoy — APPROVED** [review]

File: terra-env/main.tf (hunk 2 above)
Comment (jaycarlton) [reply terra-env/main.tf]: old habits die hard

**Pull request merged**